src/network/networkd-routing-policy-rule.c

```diff
@@ -746,6 +746,11 @@ static const RoutingPolicyRule kernel_rules[] = {
 static bool routing_policy_rule_is_created_by_kernel(const RoutingPolicyRule *rule) {
         assert(rule);
 
+        if (rule->l3mdev > 0)
+                /* Currently, [RoutingPolicyRule] does not explicitly set FRA_L3MDEV. So, if the flag
```

Comment:
What happens if this changes in the future? Are there other places we should document this in the code, to ensure they are updated concurrently in case it's needed in the future?

Reply:
Further comment is added in `routing_policy_rule_section_verify()`.